source/logo/faviconLogo.svg

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Maybe it makes more sense to put the favicon logos under `source/favicons/`. Also, feel free to put the pngs under `media.taskcluster.net/favicons/`.